docs/fixture-format.md

```diff
@@ -319,6 +319,16 @@ Then, either use the resolved channel keys directly in a mode's channel list, or
 * `"eachPixelGroup"`: Gets computed into an array of all pixel group keys, ordered by appearance in the JSON file.
   - For the above [matrix structure](#matrix-structure) example, this results in `["Inner ring", "Middle ring", "Outer ring"]`.
 
+`channelOrder` defines how the channels are ordered. Possible values are:
+
+* `"perPixel"`: For the above [matrix structure](#matrix-structure) example, this results in
+  - `["Red Inner ring", "Green Inner ring", "Blue Inner ring"]`
+  - `["Red Middle ring", "Green Middle ring", "Blue Middle ring"]`
+  - `["Red Outer ring", "Green Outer ring", "Blue Outer ring"]`
+* `"perChannel"`: For the above [matrix structure](#matrix-structure) example, this results in 
+  - `["Red Inner ring", "Red Middle ring", "Red Outer ring"]`
+  - `["Green Inner ring", "Green Middle ring", "Green Outer ring"]`
+  - `["Blue Inner ring", "Blue Middle ring", "Blue Outer ring"]`
```

Comment:
Technically, it always results in a flat list, so maybe better replace it with this?

```suggestion
* `"perPixel"`: For the above [matrix structure](#matrix-structure) example, this results in
  - `"Red Inner ring"`
  - `"Green Inner ring"`
  - `"Blue Inner ring"`
  - `"Red Middle ring"`
  - `"Green Middle ring"`
  - `"Blue Middle ring"`
  - `"Red Outer ring"`
  - `"Green Outer ring"`
  - `"Blue Outer ring"`
* `"perChannel"`: For the above [matrix structure](#matrix-structure) example, this results in
  - `"Red Inner ring"`
  - `"Red Middle ring"`
  - `"Red Outer ring"`
  - `"Green Inner ring"`
  - `"Green Middle ring"`
  - `"Green Outer ring"`
  - `"Blue Inner ring"`
  - `"Blue Middle ring"`
  - `"Blue Outer ring"`
```

However, this is quite long. Better suggestions?